migrations/2022-02-21-211645_create_sessions/up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE persistent_sessions
+(
+    id SERIAL
```

Comment:
I recommend BIGSERIAL (8 bytes instead of 4 bytes). You may not think we're likely to use up 2B entries, but it's a big hassle if and when it does happen. For example: https://community.letsencrypt.org/t/2019-11-17-autoincrement-maxed-out/106563/2

Reply:
Done.